src/v2/Components/SavedSearchAlert/SavedSearchAlertModal.tsx

```diff
@@ -170,8 +166,6 @@ export const SavedSearchAlertModal: React.FC<SavedSearchAlertFormProps> = ({
                   />
                 </Box>
               </Box>
-
-              <DownloadAppBanner entity={entity} />
```

Comment:
**praise**: Ahh, nice. I was wondering when we were going to get around to this 👍🏼